slick.grid.js

```diff
@@ -913,6 +913,91 @@ if (typeof Slick === "undefined") {
       }
     }
 
+    function sortColumnByActiveCell() {
+      var $col;
+      var activeCell;
+      activeCell = getActiveCell();
+      if (activeCell) {
+        $col = $(getHeaderColumn(activeCell.cell)).closest(".slick-header-column");
```

Comment:
this is to make sure the column definition exists, right? if so you could use `getColumns()` to see:
1. whether the column in the active cell is valid
2. whether it is sortable